doc/nrf/links.txt

```diff
@@ -689,3 +689,7 @@
 .. _`HERE Positioning`: https://www.here.com/platform/tracking-positioning-solutions/positioning
 
 .. _`Skyhook Precision Location`: https://www.skyhook.com/precision-location
+
+.. _Install chocolatey: https://chocolatey.org/install
```

Comment:
```suggestion
.. _`Install chocolatey`: https://chocolatey.org/install
```

Reply:
This was intentional. I'll check if this doesn't break the build before I commit.